Mage.Sets/src/mage/cards/i/IcebreakerKraken.java

```diff
@@ -80,9 +82,15 @@ public IcebreakerKraken copy() {
 
 class IcebreakerKrakenEffect extends OneShotEffect {
 
+    private static final FilterPermanent filter = new FilterPermanent();
+
+    static {
+        filter.add(Predicates.or(CardType.CREATURE.getPredicate(), CardType.ARTIFACT.getPredicate()));
+    }
+
     IcebreakerKrakenEffect() {
-        super(Outcome.Benefit);
-        staticText = "artifacts and creatures target opponent controls don't untap during that player's next untap step";
+        super(Outcome.Detriment);
+        staticText = "Artifacts and creatures target opponent controls don't untap during that player's next untap step";
```

Comment:
Static text for effects shouldn't be capitalized. The rule text generation will capitalize it if needed. In this case, it is after a trigger phrase so lowercase is correct.